extension/src/popup/components/sendPayment/SendSettings/Settings/index.tsx

```diff
@@ -129,29 +127,25 @@ export const Settings = ({
     }
 
     if (isSoroswap) {
-      let simulatedTx;
-      try {
-        simulatedTx = await buildAndSimulateSoroswapTx({
+      const simulatedTx = await dispatch(
```

Comment:
following the pattern we set with simulating soroban token transfers, which is already set up to show sim tx errors